### PR TITLE
[FEM] Elmer: group mechanical equations

### PR DESCRIPTION
**Pull request by donovaly**
- for consistency since the idea is that every physics group gets one entry and in future more mechanical equations might be added.

### CODE REVIEW

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9013">pipeline status</a> for feature branch [PR_9013](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9013). Pipeline [816325404](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/816325404) was triggered at [302105b](https://github.com/FreeCAD/FreeCAD/pull/9013/commits/302105b4df334178a5d4c1036868b65918c5390b). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/Mod/Fem/Gui/Command.cpp [hunk 1]
```diff
@@ -1508,6 +1508,121 @@ bool CmdFemCompEmEquations::isActive()
...
+}
+
+bool CmdFemCompMechEquations::isActive()
+{
+    // only if there is an active analysis
+    Fem::FemAnalysis* ActiveAnalysis =
+        FemGui::ActiveAnalysisObserver::instance()->getActiveObject();
+    if (!ActiveAnalysis
+        || !ActiveAnalysis->getTypeId().isDerivedFrom(Fem::FemAnalysis::getClassTypeId()))
+        return false;
```
Comment (marioalexis84) [inline src/Mod/Fem/Gui/Command.cpp:1610]: The `ActiveAnalysisObserver` class has the `hasActiveObject` function.

Comment (donovaly) [reply src/Mod/Fem/Gui/Command.cpp]: Thanks. I could this way also simplify existing code

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9013">pipeline status</a> for feature branch [PR_9013](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9013). Pipeline [816369985](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/816369985) was triggered at [713eeb2](https://github.com/FreeCAD/FreeCAD/pull/9013/commits/713eeb21caf11e681cc415bd0b2a596e6e6e2f58). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Pull request merged**